scripts/helm-test-runner.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+
+# helm-image-runner.sh contains functions used to run the ACK Helm tests
+
+SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT_DIR="$SCRIPTS_DIR/.."
+
+AWS_SERVICE=$(echo "${AWS_SERVICE:-""}" | tr '[:upper:]' '[:lower:]')
+
+DEFAULT_SERVICE_CONTROLLER_SOURCE_PATH="$ROOT_DIR/../$AWS_SERVICE-controller"
+SERVICE_CONTROLLER_SOURCE_PATH=${SERVICE_CONTROLLER_SOURCE_PATH:-$DEFAULT_SERVICE_CONTROLLER_SOURCE_PATH}
+
+HELM_DIR="$SERVICE_CONTROLLER_SOURCE_PATH/helm"
+
+source "$SCRIPTS_DIR/lib/common.sh"
+source "$SCRIPTS_DIR/lib/config.sh"
+source "$SCRIPTS_DIR/lib/logging.sh"
+
+source "$SCRIPTS_DIR/controller-setup.sh"
+
+install_chart_and_run_tests() {
```

Comment:
This feels a bit oddly coupled, to be honest. I think there should be totally separate scripts that do installation/setup and scripts that have tests in them. Recommend moving `install_chart_and_runt_tests` to the `run-helm-tests.sh` script below and renaming this file to simply `scripts/helm.sh`